### ISSUE TITLE
New source high sigma and edge sources confusion with forced photometry

### ISSUE THREAD

**Issue by ajstewart** [askap-vast/vast-pipeline#207]
TLDR: There can be confusion when a new source is classed new because it is at the edge of the image and forced photometry doesn't work for extreme edge sources.

Scenario:

We have 3 epochs, 1, 2, and 3. They overlap in area but not exactly. Epoch 1 is less sensitive than 2 and 3.

* A source is detected in epoch 3 that is not in epoch 1 or 2. In epoch 2 this source happens to be right on the edge of the field, in epoch 1 it is near the middle.
* New source analysis looks at the source in epoch 2 and is able to pull an RMS measurement by a pixel value in epoch 2, and assigns it a new source high sigma value of 8 because this is the highest sigma it would have.
* Forced extraction then occurs but is not able to extract the source in epoch 2 because it is at the edge.
* The user then looks at the results and sees that the new high sigma value is high, but only sees the forced extraction in epoch 1, which is not near this sigma value. The user can't easily see why this new source value has occurred.

An example.

The second source here has a high new source sigma of 11.056
<img width="1365" alt="Screen Shot 2020-07-15 at 17 34 52" src="https://user-images.githubusercontent.com/3306161/87516960-8b85a200-c6c1-11ea-9b9b-4110e6800b67.png">

However when looking at the source it is quite faint, and nowhere near this number in the first forced fit image (which in this case is much less sensitive. The actual image is missing in-between RACS and SB9649 because the forced fit failed.
<img width="1489" alt="Screen Shot 2020-07-15 at 17 35 02" src="https://user-images.githubusercontent.com/3306161/87517225-eae3b200-c6c1-11ea-8eee-4175b7eb5b2d.png">
<img width="1460" alt="Screen Shot 2020-07-15 at 17 41 56" src="https://user-images.githubusercontent.com/3306161/87517655-82490500-c6c2-11ea-9431-1e6ecde3c668.png">

Position of source in SB9602:
<img width="534" alt="Screen Shot 2020-07-15 at 17 40 01" src="https://user-images.githubusercontent.com/3306161/87517514-457d0e00-c6c2-11ea-820b-71d31b242902.png">

**Solution:** When performing the RMS measurement for the new source measurement the process would need to check whether the source is near the edge. However this will probably come at a cost of speed as you'll need to create slices of the array to check for NaN values.

I'll think more about how to handle this.
